pkg/volume/util/operationexecutor/operation_executor.go

```diff
@@ -500,6 +508,44 @@ func (oe *operationExecutor) generateDetachVolumeFunc(
 	}
 
 	return func() error {
+		if verifySafeToDetach {
+			// Fetch current node object
```

Comment:
is there a log line we can look for to check if a volume was detached in the case where this flag is false (so the timeout case)? if not, how hard is it to surface?